packages/@cdktf/hcl2cdk/lib/index.ts

```diff
@@ -49,24 +49,28 @@ Please include this information:
 ${JSON.stringify((err as z.ZodError).errors)}`);
   }
 
+  // Each key in the scope needs to be unique, therefore we save them in a set
+  const scope = new Set<string>();
```

Comment:
maybe also name this `scopeIdentifiers`? Sounds a bit more clear than just scope

Reply:
Yeah good idea :)